supervisor/rpcinterface.py

```diff
@@ -60,7 +60,7 @@ def _update(self, text):
     def getAPIVersion(self):
         """ Return the version of the RPC API used by supervisord
 
-        @return string version version id
+        @return string version id
```

Comment:
sometimes doubled words are intentional. I thought about this one a bit and don't think that this doubling adds anything and it doesn't seem to match other forms in this file...